src/main/java/com/google/plantasy/servlets/LogoutServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.plantasy.utils.UserUtils;
+import com.google.plantasy.utils.CookieUtils;
+
+@WebServlet("/logout")
+public class LogoutServlet extends HttpServlet {
+DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      Cookie[] cookies = request.getCookies();
+      Entity userEntity = UserUtils.getUserFromCookie(cookies, datastore);
```

Comment:
Should this be an entity or a User class?

Reply:
the user class doesnt handle session id's rn so it has to b an entity

Reply:
gotcha, I would maybe add that to the user Entity in this CL so we can make that change. We shouldnt be introducing new references to the Entity class.

Reply:
ok cool ill edit it